deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -458,7 +458,9 @@ async def create_new_job(
             job_id = await create_job(
                 con=con,
                 upload_id=details.upload_id,
-                queue=f"pulse3d-v{details.version}",
+                queue=f"test-pulse3d-v{details.version}"
```

Comment:
nitpick, I'd probably pull this if/else out of the params so it's easier to follow the logic